pyega3/pyega3.py

```diff
@@ -110,62 +122,71 @@ def get_token(credentials):
         oauth_token = reply['access_token']
         logging.info("Authentication success for user '{}'".format(username))
     except Exception as ee:
```

Comment:
It would be good to replace the name of the variable `ee` for exception, because `ee` can have more meanings than expect exception. My suggestion use `expectedException`

Reply:
Done.